Backend/app/controllers/customer.controller.js

```diff
@@ -1,5 +1,58 @@
-const Customer = require("../models/customer.model.js");
+const Customer = require('../models/customer.model');
+const CaseNote = require('../models/casenote.model');
 
 // Create and Save a new Customer
 
+const customerController = {
+  async getAllUsers(req, res) {
+    try {
+      const results = await Customer.retrieveAll();
+      res.send({ customers: results });
+    } catch (err) {
+      console.error(err);
+      // TODO error handling
+      res.status(500);
+      res.send({ error: err });
+    }
+  },
+  async getAlertCase(req, res) {
+    try {
+      const { user_id } = req.params;
+      const case_id = await Customer.getAlertCaseId(user_id);
```

Comment:
It might be better to add foreign key constraint on alert case id and do a join here instead of making two db queries